controllers/users/register_v2.js

```diff
@@ -44,6 +44,7 @@ const registerV2 = async (req, res) => {
       await checkUser.update(users);
       updatedUser = await checkUser.save();
     } catch (e) {
+      console.log('error on updating unverified user', e);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

While it's good to log errors for debugging purposes, using `console.log` is not the best practice in a production environment. It would be better to use a dedicated logging library that can provide more control over log levels, formats, and destinations. Here's an example of how you might use the `winston` library:

```diff
- console.log('error on updating unverified user', e);
+ const winston = require('winston');
+ winston.error('error on updating unverified user', e);
```
This way, you can configure Winston to log errors to a file or a remote log collection service, which can be very helpful for troubleshooting issues in a live application.


<!-- This is an auto-generated comment by OSS CodeRabbit -->